$NetBSD: patch-aa,v 1.4 2015/12/29 23:34:44 dholland Exp $

Don't use LTCONFIG_OVERRIDE and let pkgsrc automatically use the
pkgsrc-installed libtool.  Modify the configure script to grep the
libtool we're using for the values of various libtool-derived
variables.

--- configure.orig	1999-11-05 16:07:44.000000000 -0500
+++ configure
@@ -3279,14 +3279,16 @@ EOF
 
 
 
+test -z "$libtool_path" || libtool_path="./libtool"
+
 echo $ac_n "checking grepping libtool to find wl_flag""... $ac_c" 1>&6
 echo "configure:3284: checking grepping libtool to find wl_flag" >&5
 if eval "test \"`echo '$''{'cim_cv_wl_flag'+set}'`\" = set"; then
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   
-eval `grep wl= libtool`
-cim_cv_wl_flag=$pic_flag
+eval `grep ^wl= $libtool_path`
+cim_cv_wl_flag=$wl
 
 fi
 
@@ -3303,7 +3305,7 @@ if eval "test \"`echo '$''{'cim_cv_link_
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   
-eval `grep link_static_flag libtool`
+eval `grep ^link_static_flag= $libtool_path`
 cim_cv_link_static_flag=$link_static_flag
 
 fi
@@ -3321,7 +3323,7 @@ if eval "test \"`echo '$''{'cim_cv_pic_f
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   
-eval `grep pic_flag libtool`
+eval `grep ^pic_flag= $libtool_path`
 cim_cv_pic_flag=$pic_flag
 
 fi
